.travis.yml

```diff
@@ -1,13 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "iojs"
```

Comment:
multiple `node` version does not affect the build, as the test environment is browser.